apps/web-connect/src/features/checkout/Checkout.tsx

```diff
@@ -46,9 +46,9 @@ export function Checkout() {
     }
 
 	useEffect(() => {
-		if (!stakingTabOpened && mintWithEth.isSuccess || mintWithXai.isSuccess) {	
+		if (!stakingTabOpened && (mintWithEth.isSuccess || mintWithXai.isSuccess)) {	
             setStakingTabOpened(true);
-			window.open("https://app.xai.games/staking?mint=true", '_blank');
+			window.open("https://develop.app.xai.games/staking?showKeys=true", '_blank');
```

Comment:
This needs to have the `modal` query param from https://github.com/xai-foundation/sentry/pull/286

Reply:
The PR you mentioned here [#286](https://github.com/xai-foundation/sentry/pull/286) is going into develop. 

Does this PR need to be moved to develop? Also, the PR above is not merged yet.

Reply:
No, this one should be into tiny keys. #286 could go into either develop or tiny keys, its no breaking change or functinality update other than adding a query param for a modal to show up.

The staking app only gets deployed when pushed to develop, thats probably why the PRs kept going into develop. However we should put them into fb-tiny-keys if its a feature for that epic.

Reply:
This has been corrected.